assignment.js

```diff
@@ -66,6 +86,16 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const numbers = [57, 13, 4, 38]
+
+//console.log(numbers)
+
+//for (let i = numbers[])
+
+let currentLargestNumber = Math.max(...numbers)
```

Comment:
@dkwatson522 nice use of the spread operator combined with `Math.max()`. I'm all for using these advanced approaches as long as you understand how it works.